### DISCUSSION TITLE
foo.bar not found, did you mean foo.bar?

### DISCUSSION THREAD

**Discussion by muhammadmp97** [laravel/framework#43018] [Q&A] [answered]
- Laravel Version: 8.40
- PHP Version: 8.1
- Database Driver & Version: -

### Description:
When you create a view named "foo.bar.blade.php" in views directory, you see such an error:
View [foo.bar] not found... Did you mean foo.bar?
It looks both funny and confusing, but we know why it happens: Laravel replaces the dots with slashes.
Is it a bug? I'll be happy to contribute and fix it if you consider such things bugs. Hence, let me know to try.

### Steps To Reproduce:
Use dots in the view names.

**Comment by henzeb** [comment]
does the problem also persist in laravel 9.19.0, which is the latest version as of writing?

**Reply by muhammadmp97** [reply]
I started to explore and found that it has nothing to do with Laravel. It's about facade-ignition that shows us beautiful error pages!

This is the pull request I created and I hope they merge it:
https://github.com/facade/ignition/pull/457

**Reply by henzeb** [reply]
I doubt they will merge that...you didn't add a test.

**Comment by muhammadmp97** [comment, marked as answer]
Happily I fixed the problem and the pull request accepted:
https://github.com/facade/ignition/pull/457

**Reply by getsolaris** [reply]
Great
